conda/config.py

```diff
@@ -171,13 +191,11 @@ def pkgs_dir_from_envs_dir(envs_dir):
 # Note, get_default_urls() and get_rc_urls() return unnormalized urls.
 
 def get_default_urls():
-    if isfile(sys_rc_path):
-        sys_rc = load_condarc(sys_rc_path)
-        if 'default_channels' in sys_rc:
-            return sys_rc['default_channels']
-
-    return ['http://repo.continuum.io/pkgs/free',
-            'http://repo.continuum.io/pkgs/pro']
+    if 'default_channels' in sys_rc:
```

Comment:
For the record, I implemented this independently, only without the addition of the default_channels, which is superfluous (a user can add the defaults by adding a ``defaults`` channel).